bin/juxtaposer/README.md

```diff
@@ -180,6 +204,17 @@ backends:
     sslmode: disable
     debug: false
     ignore: true
+
+  mssql_secretless:
+    host: 127.0.0.1
+    ignore: true
```

Comment:
Should this have a `port: 1434` setting?

Reply:
Default port for mssql is 1433 and our secretless should be listening to that (once I fix the other two comments you mentioned) :)